packages/plugins/plugin-navtree/src/components/NavTree/NavTree.stories.tsx

```diff
@@ -40,17 +40,24 @@ faker.seed(1234);
 const StoryState = defineCapability<{ tab: string }>('story-state');
 
 // TODO(burdon): How to adjust existing surfaces for this test?
-// TODO(burdon): How to toggle attention in this test?
 // TODO(burdon): Fix outline (e.g., button in sidebar nav is clipped when focused).
 // TODO(burdon): Consider similar containment of: Table, Sheet, Kanban Column, Form, etc.
 
 const container = 'flex flex-col grow gap-2 p-4 rounded-md';
 
 // TODO(burdon): Factor out PlankHeader.
-const StoryPlankHeading = () => {
+const StoryPlankHeading = ({ attendableId }: { attendableId: string }) => {
+  const { hasAttention } = useAttention(attendableId);
+  console.log('hasAttention', hasAttention);
   return (
     <div className='flex p-1 items-center border-b border-separator'>
-      <IconButton density='coarse' icon='ph--atom--regular' label='Test' iconOnly classNames='w-[40px] h-[40px]' />
+      <IconButton
+        density='coarse'
+        icon='ph--atom--regular'
+        label='Test'
+        iconOnly
+        classNames={mx('w-[40px] h-[40px]', hasAttention && 'bg-accentSurface text-accentSurfaceText')}
```

Comment:
fwiw I think we usually just do `variant={hasAttention ? 'primary' : 'ghost'}` for the sigils; also their size is `is-[--rail-action] bs-[--rail-action]`